app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -61,20 +55,32 @@ def value_size
         size == "lg" ? 1 : 2
       end
 
+      def formatted_value
```

Comment:
This method, and perhaps more, could be made private if they are not getting used in the html.erb file.